war/src/main/js/pluginSetupWizardGui.js

```diff
@@ -250,6 +252,16 @@ var createPluginSetupWizard = function(appendTarget) {
 		}
 		return null;
 	};
+	var setFailureStatus = function(plugData) {
+		var plugFailIdx = failedPluginNames.indexOf(plugData.name);
+		if(/.*Fail.*/.test(plugData.installStatus)) {
```

Comment:
Not entirely sure, tbh. This is what other spots are using. I can refactor it later, prefer to keep changes here small due to merge mess with the upgrade wizard.